components/automate-ui/src/app/pages/project/rules/project-rules.component.html

```diff
@@ -77,26 +77,23 @@
               <chef-form-field class="attribute-field">
                 <label>
                   <span class="label">{{ getAttributeLabel() }}</span>
-                  <select ngDefaultControl formControlName="attribute" data-cy="attribute-dropdown">
-                    <option
-                      *ngFor="let option of attributes[ruleForm.get('type').value.toLowerCase()]; let i = index"
-                      [value]="option.key"
-                    >{{ option.value }}</option>
-                  </select>
+                  <chef-select ngDefaultControl formControlName="attribute" data-cy="attribute-dropdown" (change)="patchAttributeValue(ruleForm, $event, i)">
+                    <chef-option *ngIf="selectAttributeChefOption && currentRuleIndex !== i">Select...</chef-option>
```

Comment:
This (1) gets rid of two extra variables, similar to above, but also (2) fixes a bug. The currentRuleIndex could be updated frequently if there are multiple new conditions added. So that would make `Select` reappear as an option when it should not.
```
                  <chef-select ngDefaultControl #ruleAttribute formControlName="attribute" data-cy="attribute-dropdown" (change)="patchAttributeValue(ruleForm, $event, i)">
                    <chef-option *ngIf="!ruleAttribute.value">Select...</chef-option>
```